modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -1,4 +1,5 @@
 = User Authentication with LDAP
+:experimental:
```

Comment:
Shouldn't `experimental` be already part of the settings for all documents and therefore not needed on a particular page? I remember that you have implemented that one some time ago...?

Reply:
That slipped in by accident. I was using the live preview plugin to test in the browser.